packages/integration-karma/test/template/directive-for-each/index.spec.js

```diff
@@ -58,14 +58,13 @@ function iterator() {
 }
 testForEach('Iterator', { [Symbol.iterator]: iterator });
 
-// TODO [#1285]: revisit
-xit('should throw an error when the passing a non iterable', () => {
+it('should throw an error when the passing a non iterable', () => {
     const elm = createElement('x-test', { is: XTest });
     elm.items = {};
 
     // TODO [#1283]: Improve this error message. The vm should not be exposed and the message is not helpful.
     expect(() => document.body.appendChild(elm)).toThrowError(
```

Comment:
If the only difference is `Object` -> `ProxyObject`, it would be better to scope the regex to this, instead of duplicating it entirely. Something like:

```
/Invalid template iteration for value `\[object (ProxyObject|Object)\]` in \[object:vm undefined \(\d+\)\]. It must be an array-like object and not `null` nor `undefined`./
```